Formula/qt@5.5.rb

```diff
@@ -115,6 +116,9 @@ def install
     inreplace prefix/"mkspecs/qconfig.pri", /\n\n# pkgconfig/, ""
     inreplace prefix/"mkspecs/qconfig.pri", /\nPKG_CONFIG_.*=.*$/, ""
 
+    # see: https://github.com/Homebrew/homebrew-core/issues/8777
+    inreplace prefix/"mkspecs/features/mac/default_pre.prf", Regexp.escape('isEmpty($$list($$system("/usr/bin/xcrun -find xcrun 2>/dev/null")))'), 'isEmpty($$list($$system("/usr/bin/xcrun -find xcodebuild 2>/dev/null")))'
```

Comment:
Split this over multiple lines. Could simplify it with `/xcrun -find xcrun/, "xcrun -find xcodebuild"`.